### PR TITLE
Update documentation for Ubee Device Tracker component

### PR DESCRIPTION
**Pull request by mzdrale**
**Description:**

Updated documentation for Ubee Device Tracker component, which now supports mode [Ubee](http://www.ubeeinteractive.com) models.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22217

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/device_tracker.ubee.markdown
Comment (klaasnicolaas) [inline source/_components/device_tracker.ubee.markdown]: You must put the links in html style 😉 https://www.w3schools.com/html/html_links.asp

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/device_tracker.ubee.markdown
Comment (mzdrale) [reply source/_components/device_tracker.ubee.markdown]: My bad, I was in hurry and didn't see I mixed html and markdown. :)

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Can be merged after merging the parent PR 😉

**Comment by mzdrale** [comment]
Parent PR is closed.
New one will be opened.

**Pull request closed without merging**
